projects/metallb/metallb/README.md

```diff
@@ -14,8 +14,23 @@ MetalLB is a virtual-ip provider for kubernetes services of type load balancer.
 1. Review [releases notes](https://metallb.universe.tf/release-notes/)
     * Any changes to the upstream configuration needs a thorough review + testing
     * Deprecation or removal of any protocol must be considered breaking 
-1. Update the `GIT_TAG` file to have the new desired version based on the upstream release tags.
-1. Verify the golang version has not changed. 
+1. Update the `GIT_TAG` and `GOLANG_VERSION` files to have the new desired version based on the upstream release tags.
+1. Review the patches under `patches/` folder and remove any that are either merged upstream or no longer needed.
```

Comment:
What if we need to add any? Should we have a pointer on when that might be the case?

Reply:
Some of the cases where we need to add patch for package support are listed [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/docs/development/packages/update-helm-charts.md). I have linked this doc in this section in the README

`
To make changes to the patches folder, follow the steps mentioned [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/docs/development/packages/update-helm-charts.md#generate-patch-files)
`